docs/book/introduction.md

```diff
@@ -4,27 +4,178 @@ description: Welcome to ZenML!
 
 # ⭐ Introduction
 
-🤹 Are you an ML engineer or data scientist shipping models to production while juggling a plethora of tools?
+ZenML is an extensible, open-source MLOps framework for creating portable, 
+production-ready machine learning pipelines.
+By decoupling infrastructure from code, ZenML gives all developers in your 
+organization more freedom and independence in how they approach their work.
 
-🤷‍♂️ Do you struggle with versioning data, code, and models in your projects?
+![ZenML Overview](.gitbook/assets/intro_zenml_overview.png)
 
-👀 Have you had trouble replicating production pipelines and monitoring models in production?
+{% tabs %}
 
-[🙇](https://apps.timwhitlock.info/emoji/tables/unicode#emoji-modal) Are you dealing with lots of unstructured notebooks or scripts with lots of manual processes?
+{% tab title="ZenML for Data Scientists" %}
 
-[🚧](https://apps.timwhitlock.info/emoji/tables/unicode#emoji-modal) Do you find it challenging to transition your ML workflows from the experimentation setting to a production-ready environment?
+As a Data Scientist, ZenML gives you the freedom to focus on modeling and
```

Comment:
Maybe it's just me, but whenever I see 'As a ...' sentences, I think of 'As a large language model trained by OpenAI...'. Smells a bit odd, but maybe it's just me...

Reply:
@bcdurak didn't like it either, see above. I'll change it 👍